nextcord/ext/help_commands/embedded.py

```diff
@@ -0,0 +1,195 @@
+from typing import Mapping, Optional, Union
+
+from nextcord import Embed, Message
+from nextcord.ext import commands
+
+
+class EmbeddedHelpCommand(commands.HelpCommand):
+    """A help command implementation using embeds.
+    This inherits from :class:`HelpCommand`.
+    It extends it with the following attributes.
+
+    Attributes
+    ------------
+    dm_help: Optional[:class:`bool`]
+        A bool that indicates if the output should only be sent via DM instead of the channel.
+        Defaults to False.
```

Comment:
Probably doesn't need to be Optional since it defaults to false.

Same with anything that defaults to a string or number. The docstring should be updated to match the parameter types.